bundle/config/mutator/populate_current_user_test.go

```diff
@@ -1,3 +1,40 @@
 package mutator
 
-// We need to implement workspace client mocking to implement this test.
+import "testing"
+
+func TestPopulateCurrentUser(t *testing.T) {
+	// We need to implement workspace client mocking to implement this test.
+}
+
+func TestGetShortUserName(t *testing.T) {
+	tests := []struct {
+		name     string
+		email    string
+		expected string
+	}{
+		{
+			name:     "test alphanumeric characters",
+			email:    "test.user@example.com",
+			expected: "test_user",
+		},
+		{
+			name:     "test unicode characters",
+			email:    "tést.üser@example.com",
+			expected: "tést_üser",
```

Comment:
I can imagine us using these short names in UC entities as well. Can we use unicode there?

Reply:
UC supports Unicode, and all currently supported resource types support Unicode. So I'd propose to stick to Unicode  and manage by exception for any case we find where it's not supported (those cases should be fixed at the service side!)

Note that in this stage, Registered Models are the most restrictive and have been the reason I filtered out non-letter characters:
> A workspace registered model name must be a non-empty UTF-8 string and cannot contain forward slashes(/), periods(.), or colons(:). If trying to access models in the Unity Catalog, make sure to install version 2.4.1 or above of the MLflow Python client, then configure the MLflow client to access models in Unity Catalog via mlflow.set_registry_uri('databricks-uc') before accessing models.